### ISSUE TITLE
[Accessibility] ToolTip shows up in UIA tree as focusable and onscreen after dismissing and has no Name UIA property

### ISSUE THREAD

**Issue by John-Qiao** [dotnet/winforms#3885]
* .NET Core Version: 
.NET SDK 6.0.100-alpha.1.20460.31

* Have you experienced this same bug with .NET Framework?: 
Yes

**More Info:**
When setting the ToolTip to Form itself, the ToolTip can be focusable using Inspect tool, and no error shows in Accessibility Insights For Windows tool testing result. 
![Inspect-Form](https://user-images.githubusercontent.com/45864985/92889321-b46bae80-f448-11ea-8cc6-067859328558.png)
![AI-Form](https://user-images.githubusercontent.com/45864985/92889308-b170be00-f448-11ea-9b06-f3264ffa7ebe.png)

**Problem description:**
The ToolTip is not focusable when set it to the control.  
![AI-control](https://user-images.githubusercontent.com/45864985/92889333-b6ce0880-f448-11ea-80a0-07c66c7d08bb.png)

**Expected behavior:**
The ToolTip should be focusable when set it to the control.

**Minimal repro:**

1. Extract “testApp.zip” and open it in VS.
2. Build and run the project.
3. Move the mouse on button1 to shows its tooltip text.
4. Open Accessibility Insights For Windows tool to test that tooltip text of button1. 
[TestApp.zip](https://github.com/dotnet/winforms/files/5207538/TestApp.zip)

**Comment by SergeySmirnov-Akvelon** [comment]
If I understand correctly, the bug is that when you try to move cursor to the button tooltip, the tooltip disappears, though for the form it continues to be displayed. I think this happens because the tooltip is displayed while the cursor is on the element that the tooltip belongs to. If the button is small, then when you try to focus on the tooltip, the cursor goes beyond the boundaries of the button and the tooltip disappears. I tried to use big button and I was able to hover the Accessibility Insights to the tooltip and get data about it:
![3885-AI](https://user-images.githubusercontent.com/23376742/96581599-ba27a000-12e2-11eb-8121-0c042e90fd42.jpg)

The tests were also successful:
![3885-tests](https://user-images.githubusercontent.com/23376742/96581629-c7448f00-12e2-11eb-950c-7c6fbfefb48c.jpg)

**Comment by John-Qiao** [comment]
@SergeySmirnov-Akvelon yes, when the tooltip text is completely 'included' in the button control, the tooltip text can be focused and no error shows in Accessibility Insights tool testing result. If the tooltip text is partially 'included' in the button control, it can be focused using Accessibility Insights tool too, but the initial error still shows in Accessibility Insights tool testing result. Please check below gif file:
![TestResult](https://user-images.githubusercontent.com/45864985/96662208-69ff1b00-1380-11eb-87a7-63023d85763f.gif)

**Comment by SergeySmirnov-Akvelon** [comment]
We think that current behavior is expected for tooltip. The moment the tooltip disappears, it stops returning its name. Unfortunately it is behavior of native control by default.
![3885-NativeControl](https://user-images.githubusercontent.com/23376742/96693286-b9007c80-138f-11eb-8b9f-7e459d53ff6b.gif)
If we look in the Inspector, and get the form components again after disappearing of tooltip then we will see that it is not displayed in the component tree
![3885-Control](https://user-images.githubusercontent.com/23376742/96693372-d6cde180-138f-11eb-85de-1a21fbdf480e.gif)
This issue does not affect the Narrator, as it annotates the tooltip text during appearing the tooltip. To change the current behavior of "Name" property we will have to add our own AccessibleObject for the tooltip, which will take time and require stabilization. @merriemcgaw could you please share your opinion?

**Comment by merriemcgaw** [comment]
Closing this issue as by design. This is how the native tooltip and at this point we don't have a way to manage this behavior. We should list this in what our requirements are for the Common Controls team when they enable WCAG2.1 support in the tooltip control.
